exercises/practice/yacht/.approaches/if-structure/content.md

```diff
@@ -0,0 +1,57 @@
+# If structure
+The constants can be set to random, null, or numeric values, and an `if` structure inside `score` determines the code to be executed. 
```

Comment:
```suggestion

The constants here can be set to random, null, or numeric values, and an `if` structure inside the `score` function can determine the code to be executed. 
```

Reply:
Is the blank line part of spec?

Reply:
Nope.  But IMHO, it reads better on the site.  However, I'm fine if you want to delete it.  😄